OPAL/tac/src/main/scala/org/opalj/tac/fpcf/analyses/alias/pointsto/TypePointsToBasedAliasAnalysisState.scala

```diff
@@ -0,0 +1,14 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
```

Comment:
Please combine these small files into larger ones in a sensible way. E.g., this code should be in the file above.